CalibPPS/ESProducers/plugins/CTPPSInterpolatedOpticalFunctionsESSource.cc

```diff
@@ -28,19 +29,26 @@ class CTPPSInterpolatedOpticalFunctionsESSource : public edm::ESProducer {
 private:
   edm::ESGetToken<LHCOpticalFunctionsSetCollection, CTPPSOpticsRcd> opticsToken_;
   edm::ESGetToken<LHCInfo, LHCInfoRcd> lhcInfoToken_;
+  edm::ESGetToken<LHCInfoPerLS, LHCInfoPerLSRcd> lhcInfoPerLSToken_;
+  edm::ESGetToken<LHCInfoPerFill, LHCInfoPerFillRcd> lhcInfoPerFillToken_;
```

Comment:
if my comment bellow works, these are to be `const`-s

Reply:
See https://github.com/cms-sw/cmssw/pull/42515#discussion_r1291759585